core/db/entities.py

```diff
@@ -309,12 +308,10 @@ class Project(BaseModel):
 
     project_name = sqla.Column(sqla.String(), nullable=False)
     primary_intervention_theme = sqla.Column(sqla.String(), nullable=False)
-    location_multiplicity = sqla.Column(
-        sqla.Enum(const.MultiplicityEnum, name="project_location_multiplicity"), nullable=True
-    )
+    location_multiplicity = sqla.Column(sqla.String, nullable=True)
     locations = sqla.Column(sqla.String, nullable=False)
     postcodes = sqla.Column(sqla.String, nullable=True)
-    gis_provided = sqla.Column(sqla.Enum(const.YesNoEnum), nullable=True)
+    gis_provided = sqla.Column(sqla.String, nullable=True)
```

Comment:
Do we not need to generate a migration for these changes??

Reply:
working on it

Reply:
Done